File: ejercicios6/empleado.php
<?php
class empleado{
    public $nombre;
    public $sueldo;

    public function mostrardetalles(){
        echo"el nombre del empleado ",$this->nombre, "y su sueldo es ",$this->sueldo,"\n";
    }
}

class gerente extends empleado{
    public $departamento;

    public function mostrardetalles(){
        echo"el nombre del empleado ",$this->nombre, ", su sueldo es ",$this->sueldo," y pertenece al departamento",$this->departamento,"\n";
    }
}

$empleado= new empleado;
$empleado->nombre="paco";
$empleado->sueldo="1650€";
$empleado->mostrardetalles();
$gerente= new gerente;
$gerente->nombre="jose";
$gerente->sueldo="3465€";
$gerente->departamento="financias";
$gerente->mostrardetalles();

File: ejercicios4/ejercicio34/src/ejercicio34/Empleado.java
package ejercicio34;

public class Empleado {
	public String nombre;
	private double salario;
	protected String departamento;
	
	public Empleado(String nombre,double salario,String departamento){
		this.nombre = nombre;
		this.salario=salario;
		this.departamento=departamento;
	}
	
	public void mostrarDatos() {
		System.out.println(nombre);
		System.out.println(salario);
		System.out.println(departamento);
	}
	 public double getSalario() {
	        return salario;
	 }
}
